auth-web/src/App.vue

```diff
@@ -38,7 +38,7 @@
       <pay-system-alert />
     </div>
     <div class="app-body">
-      <router-view  />
+      <router-view :key="$route.path" />
```

Comment:
what is this for?

Reply:
Basically if we don't have this and you navigate to the same URL it wont re-render the components. 

EX. navigate to:
https://test.bcregistry.ca/business/auth/account/1999/settings/account-info

but then you go into 

![image](https://user-images.githubusercontent.com/3484109/211398810-3bd257d5-888b-4437-b95e-f40139f4ec28.png)

It wont rerender the component at all, even though the route has changed.

Reply:
If I navigated repeatedly using the URL, it would work correctly.. but the button press above.. wouldn't rerender the component.